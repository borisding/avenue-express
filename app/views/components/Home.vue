<template>
  <article class="message is-info">
    <div class="message-header">
      <p>Greetings Widget</p>
      <button class="delete" aria-label="delete"></button>
    </div>
    <div class="message-body">
      <span class="greetings">{{message}}</span>
    </div>
  </article>
</template>

<script>
export default {
  data() {
    return {
      message: 'Hi! This widget is loaded from Home Vue single file component.'
    };
  }
};
</script>

<style>
.greetings {
  color: #00f;
}

.message {
  margin-top: 25px;
}
</style>