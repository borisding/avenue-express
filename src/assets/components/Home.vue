<template>
  <div class="greetings">{{ greetings }}</div>
</template>

<script>
export default {
  data() {
    return {
      greetings: 'Hi! This is loaded from Vue Home single file component.'
    };
  }
};
</script>

<style>
.greetings {
  color: #00f;
}
</style>