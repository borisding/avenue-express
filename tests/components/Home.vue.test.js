import { shallow } from '@vue/test-utils';
import Home from '@components/Home.vue';

describe('Home.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallow(Home);
    expect(wrapper).toMatchSnapshot();
  });
});
