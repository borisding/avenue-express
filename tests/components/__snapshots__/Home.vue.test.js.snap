// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue renders correctly 1`] = `
<article class="message is-info">
  <div class="message-header">
    <p>Greetings Widget</p> <button aria-label="delete" class="delete"></button>
  </div>
  <div class="message-body"><span class="greetings">Hi! This widget is loaded from Home Vue single file component.</span></div>
</article>
`;
